### DISCUSSION TITLE
Docs: Search bar nonfunctional

### DISCUSSION THREAD

**Discussion by xathon** [authelia/authelia#4359] [General]
### Description

The search bar on https://www.authelia.com doesn't appear to do anything, making it harder to search through the documentation.

Console output says `Uncaught ReferenceError: FlexSearch is not defined`.

Edit: This only happens on Chromium browsers, both mobile and Desktop.

### Reproduction

- open https://www.authelia.com
- enter anything into the search bar

**Comment by james-d-elliott** [comment]
The bug report is clearly meant for the authelia binary, please don't use it for this again.

Your reproduction instructions are invalid. Provide more information.

**Reply by xathon** [reply]
Apologies, I missed that.

I really don't know what I can do to make the reproduction instructions more complete. If I go to https://www.authelia.com and enter a term (e.g 'docker') into the search bar and press Enter, I just end up on the main page again, and there are no search results. 

The `Ctrl + /` hint doesn't do anything either.

I just checked a bit more, and it appears that this only occurs on Chromium browsers. On Safari and Firefox, the search bar works fine.

**Reply by james-d-elliott** [reply]
Strange, it used to work fine on Chrome as well and I'm fairly certain nothing changed to affect this. I can reproduce it on Chrome now.

**Reply by james-d-elliott** [reply]
Should be fixed

**Reply by xathon** [reply]
It is, thanks!
